apps/web/pages/api/email.ts

```diff
@@ -48,7 +48,7 @@ const handler = async (req: NextApiRequest, res: NextApiResponse) => {
     location: "Zoom video",
     destinationCalendar: null,
     hideCalendarNotes: false,
-    uid: "bwPWLpjYrx4rZf6MCZdKgE",
+    uid: "xxyPr4cg2xx4XoS2KeMEQy",
```

Comment:
what is this?

Reply:
https://github.com/calcom/cal.com/blob/8ff08d2c9fbdf04c9b1fb2d3245255cfacd5ad9f/apps/web/pages/api/email.ts#L7

For testing purposes only, to be able to see an email with its format without having to send it, very handy, well thought @zomars